arclight/lib/arclight/traject/ead2_component_config.rb

```diff
@@ -0,0 +1,340 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'traject'
+require 'traject/nokogiri_reader'
+require 'traject_plus'
+require 'traject_plus/macros'
+require 'arclight/level_label'
+require 'arclight/normalized_date'
+require 'arclight/normalized_component_title'
```

Comment:
Only comment I have is that maybe it makes sense to explicitly name these custom classes with an OAC namespace, or a CDL namespace, or something so it easy to know which ones are custom? This looks like it's just another Arclight core provided class unless you look at the diff. What do you think?